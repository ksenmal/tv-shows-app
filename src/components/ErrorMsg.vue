<template>
  <div class="error-msg">
    Sorry, an error has occured. Please try again later.
  </div>
</template>

<style scoped>
.error-msg {
  height: 20rem;
  text-align: center;
  margin: 2rem;
  font-size: 1.5rem;
}
</style>
