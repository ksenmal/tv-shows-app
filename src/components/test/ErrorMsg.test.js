import { mount } from "@vue/test-utils";
import ErrorMsg from "../ErrorMsg.vue";
import { describe, expect, test } from "vitest";

describe("ErrorMsg suite", () => {
  test("Snapshot", () => {
    const wrapper = mount(ErrorMsg);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
