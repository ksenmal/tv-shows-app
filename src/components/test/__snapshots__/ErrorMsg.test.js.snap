// Vitest Snapshot v1

exports[`ErrorMsg suite > Snapshot 1`] = `"<div class=\\"error-msg\\" data-v-a463373e=\\"\\"> Sorry, an error has occured. Please try again later. </div>"`;
